### PR TITLE
Issue 166 json formatted logs charts support

### PR DESCRIPTION
**Pull request by brendanjmb**
Thanks for submitting a PR! Please check the boxes below:

- [x] I have filled in the "Changes" section below?
- [x] I have filled in the "How did you test this code" section below?
- [x] I have bumped the version number in `/charts/flagsmith/Chart.yaml` in the section `version` or I'm merging to a
      release branch

## Changes

Adding ability to specify LOG_FORMAT environment variable via Helm values, in relation to issue 166 and associated PR for LOG_FORMAT environment variable to control logging behaviour. Further docs PR to follow. 

## How did you test this code?

Manually;

- Inspected helm template with the value set, and without. Ensured correct linting. 
- Applied template to K8s cluster with empty value set
- Upgraded release with values set to 'json' - verified environment variables set correctly

### CODE REVIEW

File: charts/flagsmith/values.yaml [hunk 1]
```diff
@@ -124,6 +126,8 @@ frontend:
     timeoutSeconds: 10
   extraInitContainers: []
   extraContainers: []
+  logging: {}
+    # format: json # Can be json or generic (plaintext).
```
Comment (matthewelwell) [inline charts/flagsmith/values.yaml:130]: I'm not entirely sure this is true. I don't think we've added any logic for changing the logging format in the FE code?

File: charts/flagsmith/values.yaml [hunk 2]
```diff
@@ -76,6 +76,8 @@ api:
     enabled: false
   extraInitContainers: []
   extraContainers: []
+  logging: {}
+    # format: json # Can be json or generic (plaintext).
```
[the remark this answers is not in the document]
Comment (brendanjmb) [reply charts/flagsmith/values.yaml]: @matthewelwell good shout thanks - updated.

File: charts/flagsmith/values.yaml (hunk 1 above)
Comment (brendanjmb) [reply charts/flagsmith/values.yaml]: 👍 my mistake - removed frontend references from chart & docs

**Review by matthewelwell — APPROVED** [review]
Approving but cannot be merged until [PR with the core logic](https://github.com/Flagsmith/flagsmith/pull/3376) is merged and released.

**Comment by matthewelwell** [comment]
@brendanjmb also need another version bump here, but let's prioritise getting the [PR against the main repo fixed](https://github.com/Flagsmith/flagsmith/pull/3376).

**Pull request merged**